features/admin.users.v1/pages/users.tsx

```diff
@@ -702,7 +703,13 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
                 totalPages={ resolveTotalPages() }
                 totalListSize={ usersList?.totalResults }
                 paginationOptions={ {
-                    disableNextButton: !isNextPageAvailable
+                    showItemsPerPageDropdown:
+                        (
+                            selectedUserStore === userstoresConfig.primaryUserstoreName
+                            && isSAASDeployment
+                        )
+                            ? true
+                            : false
```

Comment:
```
showItemsPerPageDropdown: selectedUserStore === userstoresConfig.primaryUserstoreName
                    && isSAASDeployment
```

Reply:
This was an existing logic. Improved with 865aeed4bb04454b2f6380b4419c93273b71d1ef.